src/aliceVision/dataio/ImageFeed.cpp

```diff
@@ -224,13 +225,8 @@ ImageFeed::FeederImpl::FeederImpl(const std::string& imagePath, const std::strin
       folder = bf::path(imagePath).parent_path().string();
       ALICEVISION_LOG_DEBUG("filePattern: " << filePattern);
       std::string regexStr = filePattern;
-      // escape "."
       boost::algorithm::replace_all(regexStr, ".", "\\.");
-      // recognize # as a digit
-      boost::algorithm::replace_all(regexStr, "#", "[0-9]");
-      // recognize @ as a sequence of digits
-      boost::algorithm::replace_all(regexStr, "@", "[0-9]+");
-      re = regexStr;
+      re = filterToRegex(regexStr);
```

Comment:
There is a boost::algorithm::replace_all done outside to remove.

Reply:
Fixed in https://github.com/alicevision/AliceVision/pull/806/commits/f7584167ca69f68bb430cb92284b5733df155fd7 .